### PR TITLE
make it so that the benchmark has a name in the tearsheet (the name o…

### PR DESCRIPTION
**Pull request by grzesir**
…f the actual asset)

### CODE REVIEW

File: lumibot/tools/indicators.py [hunk 1]
```diff
@@ -649,13 +650,13 @@ def generate_buysell_plotly_text(row):
...
-    risk_free_rate,
+    strategy_df: pd.DataFrame,
+    strat_name: str,
+    tearsheet_file: str,
+    benchmark_df: pd.DataFrame,
+    benchmark_asset: Asset,
+    show_tearsheet: bool,
+    risk_free_rate: float,
 ):
     print("Creating tearsheet...")
```
[the remark this answers is not in the document]
Comment (davidlatte) [reply lumibot/tools/indicators.py:661]: @korbit-ai-mentor has a good suggestion here I think!

File: lumibot/tools/indicators.py [hunk 2]
```diff
@@ -713,6 +714,9 @@ def create_tearsheet(
         logging.error("Not enough data to create a tearsheet, at least 2 days of data are required. Skipping")
         return
 
+    # Set the name of the benchmark column so that quantstats can use it in the report
+    df_final["benchmark"].name = str(benchmark_asset)
+
```
Comment (davidlatte) [inline lumibot/tools/indicators.py:719]: Seems to me that the "title" would be a nicer place to put the benchmark name, but I'm having a hard time visualizing what the final output looks like.

**Review by davidlatte — APPROVED** [review]
Minor thing only. Merge when ready!

File: lumibot/tools/indicators.py (hunk 2 above)
Comment (grzesir) [reply lumibot/tools/indicators.py:719]: this is just the way that the quant stats library works, unfortunately we dont have a choice here

**Pull request merged**